controllers/addon_controller.go

```diff
@@ -194,6 +198,11 @@ func (r *AddonReconciler) processAddon(ctx context.Context, req reconcile.Reques
 	// Resources list
 	instance.Status.Resources = make([]addonmgrv1alpha1.ObjectStatus, 0)
 
+	//set ttl starttime if it is 0
+	if instance.Status.TTLTime == 0 {
+		instance.Status.TTLTime = common.GetCurretTimestamp()
```

Comment:
Is this `TTLTime` really start time as we discussed yesterday? The current timestamp I see doesn't really map to TTL, I think we should rename this status to `.StartTime`?

Reply:
yes. TTLTime is startTime.  I prefer to have it starttime.  I used starttime at very beginning. I saw you does not like it when we discuss. Let me change back.